site/src/github/comparison_summary.rs

```diff
@@ -197,52 +218,25 @@ fn write_metric_summary(
             .push_str("This benchmark run did not return any relevant results for this metric.\n");
         false
     } else {
-        let primary_short_summary = generate_short_summary(&primary);
-        let secondary_short_summary = generate_short_summary(&secondary);
-
-        if hidden {
+        if !hidden {
```

Comment:
nit: Maybe `hidden` should be renamed, because now the distinction is basically reliable vs unreliable?